editor/filesystem_dock.cpp

```diff
@@ -1149,17 +1149,22 @@ void FileSystemDock::_try_move_item(const FileOrFolder &p_item, const String &p_
 		}
 
 		// Update scene if it is open.
+        int current_tab = editor->get_current_tab();
```

Comment:
Hmm, the indentation is off here and the following code in this file

Reply:
Odd, sorry about that! Formatted it and squashed it.